WhaleSpotting/react-app/src/styles/Home.scss

```diff
@@ -0,0 +1,142 @@
+@import "./constants.scss";
+
+.home {
+    padding-bottom: $footerHeight;
+    margin-top: $headerHeight;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: $secondaryLight;
+    font-family: $fontFamily;
+    margin: 0;
+    padding: 0;
```

Comment:
These are overriding line 4 and 5, do we need them all?

Reply:
Done